config/default/repository.conf.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
This file is not required as it is already registered, see `manifest > install > php`
You can choose this solution (default config files) or install by script.